dotCMS/src/main/webapp/html/js/dojo/custom-build/dijit/themes/dijit.css

```diff
@@ -1397,8 +1397,9 @@ Main class hierarchy:
     z-index: 1;
 }
 .dijitTabListWrapper {
+	width: 100% !important;
```

Comment:
What is this form? this could lead to issues because the width is assigned by the library inline in the css after calculations.

Reply:
That width is added without considering the extra padding that adds the scrollbar which makes the child div overflow the father's div width which causes an overflow-x. In order to avoid that, I set `width: 100% !important`

<img width="1194" alt="Screen Shot 2021-09-16 at 3 23 09 PM" src="https://user-images.githubusercontent.com/72418962/133672924-bf3b0244-ee5a-4f4e-8782-dbc78f33504c.png">

Reply:
We can remove the `width: 100% !important;` and add `overflow-x: hidden;` to the container.